pkg/helmClient/client.go

```diff
@@ -485,3 +487,125 @@ func copyRollbackOptions(chartSpec *ChartSpec, rollbackOptions *action.Rollback)
 	rollbackOptions.Recreate = chartSpec.Recreate
 	rollbackOptions.Wait = chartSpec.Wait
 }
+func (c *HelmClient) TemplateChart(spec *ChartSpec, options *HelmTemplateOptions) ([]byte, error) {
+	client := action.NewInstall(c.ActionConfig)
+	mergeInstallOptions(spec, client)
+
+	client.DryRun = true
+	client.ReleaseName = spec.ReleaseName
+	client.Replace = true // Skip the name check
+	client.ClientOnly = true
+	client.IncludeCRDs = true
+
+	if options != nil {
+		client.KubeVersion = options.KubeVersion
+		client.APIVersions = options.APIVersions
+	}
+
+	// NameAndChart returns either the TemplateName if set,
+	// the ReleaseName if set or the generatedName as the first return value.
+	releaseName, _, err := client.NameAndChart([]string{spec.ChartName})
+	if err != nil {
+		return nil, err
+	}
+	client.ReleaseName = releaseName
+
+	if client.Version == "" {
+		client.Version = ">0.0.0-0"
```

Comment:
is this version correct

Reply:
this line of code was mentioned in the helm client in golang docs, tested it working fine.